### PR TITLE
add product dag

### PR DESCRIPTION
**Pull request by pindge**
- takes `product_definition_uri` and `product_name`
- run `datacube product add {{uri}}`
- add it to explorer.

### CODE REVIEW

File: dags/utility/utility_add_product.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+with dag:
+    INDEXING = KubernetesPodOperator(
+        namespace="processing",
+        image=INDEXER_IMAGE,
+        image_pull_policy="IfNotPresent",
+        labels={"step": "datacube-product-add"},
+        cmds=["datacube"],
+        arguments=[
+            # "s3://dea-public-data/cemp_insar/insar/displacement/alos//**/*.yaml",
+            # "cemp_insar_alos_displacement",
```
Comment (Kirill888) [inline dags/utility/utility_add_product.py:94]: cut and paste from `dataset add`?

Comment (pindge) [reply dags/utility/utility_add_product.py:94]: yes!!. better clean it up!!!

File: dags/utility/utility_add_product.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+#### Utility customisation
+The DAG can be parameterized with run time configuration `product_definition_uri`
+
+dag_run.conf format:
+
+#### example conf in json format
+
+    "product_definition_uri": "https://raw.githubusercontent.com/GeoscienceAustralia/dea-config/master/products/lccs/lc_ls_c2.odc-product.yaml",
+    "product_name": "lc_ls_landcover_class_cyear_2_0"
```
Comment (Kirill888) [inline dags/utility/utility_add_product.py:13]: does it really need `product_name`? that is read from the yaml, and yaml might contain several products also.

Comment (pindge) [reply dags/utility/utility_add_product.py:13]: its for the second part, for explorer to update.

Comment (Kirill888) [reply dags/utility/utility_add_product.py:13]: I see it's for explorer.

**Review by Kirill888 — APPROVED** [review]

**Review by tebadi — APPROVED** [review]

**Pull request merged**